src/main/java/com/coveros/selenified/utilities/Point.java

```diff
@@ -0,0 +1,27 @@
+package com.coveros.selenified.utilities;
+
+public class Point<X, Y> {
```

Comment:
it should always be some kind of number, but there is a point to making it generic, namely, this can be re-used in the future. For clicking at points, in addition to drawing. One takes ints, the other can take doubles or ints